sw/airborne/firmwares/rotorcraft/guidance/guidance_h_ref.h

```diff
@@ -49,29 +50,20 @@ extern float gh_max_speed;
 #define GUIDANCE_H_REF_MAX_ACCEL 5.66
 #endif
 
-/** Update frequency
- */
-#define GH_FREQ_FRAC 9
-#define GH_FREQ (1<<GH_FREQ_FRAC)
 
-#define GH_ACCEL_REF_FRAC 8
-#define GH_SPEED_REF_FRAC (GH_ACCEL_REF_FRAC + GH_FREQ_FRAC)
-#define GH_POS_REF_FRAC (GH_SPEED_REF_FRAC + GH_FREQ_FRAC)
+#define GH_POS_REF_FRAC 26
```

Comment:
I think you should add a comment to explain this value. Also, as I mentioned before, the old `GH_FREQ_FRAC` of 9 is probably an implicit factor corresponding to the 512Hz, which is maybe not relevant anymore since you have introduced the real `dt`. Do we have a risk of overflow now, if the frequency is not 512 ?

Reply:
Thanks, I added a comment with the range and resolution, taking into account the frequency. I think we're pretty safe on both sides. Actually, the range is a bit overkill, more than the circumference of the earth... will update for better resolution!